tests/new_unit_tests/Commands/Connection/InfoCommandTest.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests\Commands\Connection;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+use Pantheon\Terminus\Commands\Connection\InfoCommand;
+use Pantheon\Terminus\Config;
+
+use Prophecy\Prophet;
+use Terminus\Models\Environment;
+use Terminus\Models\Site;
+use VCR\VCR;
+
+/**
+ * Test suite for class for Pantheon\Terminus\Commands\Connection\InfoCommand
+ */
+class InfoCommandTest extends ConnectionCommandTest
+{
+    private $prophet;
+
+    /**
+     * Test suite setup
+     *
+     * @return void
+     */
+    protected function setup()
+    {
+        parent::setUp();
+
+        // TODO: (ajbarry) Where is the appropriate place to configure VCR mode?
+        // Picking up 'new_episodes' somewhere
+        VCR::configure()->setMode('none');
```

Comment:
For some reason VCR started regenerating the fixtures. This will be removed once I can determine how to set this configuration in the right context (default none mode).